.env.example

```diff
@@ -1,12 +0,0 @@
-PUBLIC_APPWRITE_COL_MESSAGES_ID=
-PUBLIC_APPWRITE_COL_THREADS_ID=
-PUBLIC_APPWRITE_DB_MAIN_ID=
-PUBLIC_APPWRITE_FN_TLDR_ID=
-PUBLIC_APPWRITE_ENDPOINT=
-PUBLIC_APPWRITE_PROJECT_ID=
-PUBLIC_APPWRITE_PROJECT_INIT_ID=
-PUBLIC_GROWTH_ENDPOINT=
-APPWRITE_DB_INIT_ID=
-APPWRITE_COL_INIT_ID=
-APPWRITE_API_KEY_INIT=
-SENTRY_AUTH_TOKEN=
```

Comment:
How come these were removed?

Reply:
I accidentally deleted them when I was setting my local environment variable file for my local repo. I undid this mistake with my recent commit.